Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2239,12 +2239,10 @@ void RenderStyle::setContent(const String& string, bool add)
     auto& data = m_nonInheritedData.access().miscData.access();
     if (add && data.content) {
         auto& last = lastContent(*data.content);
-        if (!is<TextContentData>(last))
+        if (auto* textContent = dynamicDowncast<TextContentData>(last))
+            textContent->setText(textContent->text() + string);
+        else
```

Comment:
I removed this code, please make sure you undo this change when you rebase.

Reply:
Thanks for the heads-up. Rebased.